.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+# This workflow is mostly copied from the official PyPA guide:
+# https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+#
+# Note: There are improvements that could be made to streamline the release process further such as integrating helpers
+#       like bump-my-version and towncrier
+#
+# Instructions:
+#
+# 1. Finalise development on the master branch
+# 2. Update the version number in setup.py appropriately
+# 3. Commit `git commit -m "vx.x.x release"`
+# 4. Tag `git tag vx.x.x`
+# 5. Push `git push && git push --tags`
```

Comment:
I assume just adding the tag in the github UI would also achieve this too?

Reply:
Yeah steps 4+5 could be done in the GH UI but may as well do it alongside the other local steps of 2+3 🤷🏾